scripts/imm_utils.sh

```diff
@@ -41,3 +46,18 @@ remove_resources() {
     echo "Tests failed. Error occurred during ${error_message} test" 
     exit 1
 }
+
+# This function reads the variable name $1 where tenant name is/will be stored.
+# If variable with name $1 is empty, it will read the input or store default value in that variable.
+# It it's not empty, it will not change it's value 
+get_tenant_name() {
```

Comment:
This functions forces user to enter a name in every case, right? If so, I think there should be an option to pass everything from the CLI.

Reply:
Actually, this function will not force user to enter a name if it was specified. It takes the variable name, and asks for tenant name only if the variable is empty. So you can pass everything from CLI - this is how our smoke tests work.